TimeManagementSystem/TimeManagementSystem.Api2/app/dataservice.factory.js

```diff
@@ -29,13 +29,92 @@
             })
             .then(function (data) {
                 localStorage.setItem("token", "Bearer " + data.data.access_token);
-                $location.path("/home");
+                $state.transitionTo('app.home');
             })
             .catch(function (data) {
                 var errorDescription = data.data.error_description;
                 toastr.error(errorDescription);
             });
         }
+
+        function register(login, password) {
+            return $http({
+                method: 'POST',
+                url: '/api/Users',
+                data: {
+                    Login: login,
+                    Password: password,
+                    PermissionLevel: 0
+                }
+            })
+            .then(function (data) {
+                toastr.success("User created");
+                $state.transitionTo('anonymous.login');
+            })
+            .catch(function (data) {
+                var errorDescription = data.data.ModelState[""][0];
```

Comment:
W takim formacie zewnętrzna autoryzacja, której używam zwraca błędy. Tak czy siak pewnie czymś to owinę, aby było ładniej.